frontend/public/components/utils/horizontal-nav.tsx

```diff
@@ -201,7 +202,16 @@ export const HorizontalNav: React.FC<HorizontalNavProps> = React.memo((props) =>
     (extraObjs, key) => ({ ...extraObjs, [key]: _.get(props[key], 'data') }),
     {},
   );
-  const pages = props.pages || props.pagesFor(_.get(props.obj, 'data'));
+  const pages = (props.pages || props.pagesFor(_.get(props.obj, 'data')))
+    // FIXME(alecmerdler): Handle `flags` for extension tabs...
```

Comment:
This can more easily be handled later by using the hoc or hook from https://github.com/openshift/console/pull/3383